File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int[] num = new int[nums.length];
        for (int i = 0; i<nums.length; i++) {
            int res = 1;
            for (int j = 0; j<nums.length; j++) {
                if (j == i) {
                    continue;
                }
                res *= nums[j];
            }
            num[i] = res;
        }
        return num;
    }
}  

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0
        res = 0
        store = set(nums)

        for num in nums:
            streak = 0
            while num + 1 in store:
                streak += 1
                num += 1

            if streak > res:
                res = streak
        return res + 1